app/models/reaction.rb

```diff
@@ -81,7 +81,7 @@ def skip_notification_for?(receiver)
   private
 
   def touch_user
-    Users::TouchJob.perform_later(user_id)
+    Users::TouchWorker.perform_later(user_id)
```

Comment:
Given the simplicity of this job, and that we will already have the user loaded for the reaction I think it would be better to remove the job completely and simply do the `user.touch` here inline. This will prevent us from having to make multiple Redis calls and reload the user again in the job.